### ISSUE TITLE
About generate function

### ISSUE THREAD

**Issue by shanhaidexiamo** [rishikksh20/SoundStorm-pytorch#6]
Hi, thanks for your repo. I want to ask you about the generate function. You use `nn.Parameter(torch.zeros(num_codes_with_mask + 2))` , which means that the output is 1027 dim in Soundstorm.py, line 111. But in generate function, line 329, you use `logits = logits.view(batch_size, num_latents_to_generate + num_prompt, 8, 1025)` to reshape the logits to 1025 dim, so these two points are contradictory. Should the latter be changed to the 1027 dim？

Thanks
